website/docs/server_sdk.md

```diff
@@ -9,9 +9,101 @@ This page documents how to use UPS
+
+`npm i unifiedpush-node-sender`
+
+#### Usage
+
+The sender is created in your application with the following, where `url`, `applicationId`, `masterSecret` are values from your _UnifiedPush Server_
+
+```Javascript
+const agSender = require('unifiedpush-node-sender');
+
+const settings = {
+    url: '<yourPushServerURL e.g http://localhost:9999/>',
+    applicationId: '<yourPushApplicationId e.g. B868CC08-BCC8-4A0A-B21E-1AC56AF0C734>',
+    masterSecret: '<yourMasterSecret e.g. 4L30AV41-3278-4983-8F99-0EEA138J7O1I>'
+};
+```
+
+Sending a message involves first, getting a handle on the client object and using `client.sender.send` method to send a single message. 
+
+```Javascript
+agSender(settings).then((client) => {
+    client.sender.send(message, options).then((response) => {
+        console.log('success', response);
+    })
+});
+```
+
+Sending multiple messages is a similar process but instead uses the `sendBatch` method
+to pass an array of `{message, options}` objects.
+
+```Javascript
+agSender(settings).then((client) => {
+    client.sender.sendBatch(messages).then((response) => {
+        console.log('success', response);
+    })
+});
+```
+
+#### API Documentation
+
+The Sender class returns a promise with the `client` object which contains a `settings` object for the particular client. The Sender settings consists of the:
+
+- `url`  String - The URL of the Unified Push Server.
+- `applicationId` String - The ID of an Application from the Unified Push Sever
+- `masterSecret` String - The master secret for that Application
+- `headers` Object - The hash of custom HTTP headers/ header overrides 
+
+
+The `message` object that gets sent is made up of the following parameters:
+
+- `alert` String - message that will be displayed on the alert UI element
+- `priority` String - sets a processing priority on a push message, values can be 'NORMAL' or 'HIGH'
+- `sound` String - The name of a sound file
+- `badge` String - The Number to display as the badge of the app icon
+- userData Object - any extra user data to be passed 
+
+For Applications on Apple devices/products there is a `message.apns` Object
+
+- `title` String - Describes the purpose of the notification
+- `action` String - The label of the action button
+- `urlArgs` Array - An array of values that are paired with the placeholders inside the url FormatString value of your website.json file. Safari Only
+- `titleLockey` String - The key to a title string in the Localizable.strings file for the current localization. iOS Only 
+- `titleLocArgs` Array - Variable strings values appear in place of the format specifiers in title-loc-key. iOS Only
+- `actionCategory` String - The Identifier of the action category for the interactive notification
+- `contentAvailable` Boolean - Provide this ket with a value of 1 to indicate that new content is available. iOS Only
+
+The `options` Object is made up of two sub parts, the `options.config` Object and the `options.criteria` Object
```

Comment:
```suggestion
The `options` Object is made up of two sub parts, the `options.config` Object and the `options.criteria` object
```

Reply:
should the `options.config` Object capitalisation be changed to lower case o, also?

Reply:
yeah